### ISSUE TITLE
Environment "aligned" prevent numbering with autoNumber set to "all"

### ISSUE THREAD

**Issue by luc-j-bourhis** [mathjax/MathJax#1811]
Consider

```html
<!DOCTYPE html>
<html>
<head>
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "all" } }
});
</script>
<script type="text/javascript" async
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML">
</script>
</head>
<body>
<p>An equation is $$m^2=E^2-p^2$$</p>
<p>which breaks into
$$\begin{aligned}
E &= \gamma m\\
p &= \gamma m v
\end{aligned}$$
</body>
</html>
```

The first $$...$$ is numbered but not the second one.

**Comment by dpvc** [comment]
Thanks for the report.  I confirm the issue and will look into it.

**Comment by dpvc** [comment]
This turns out to be a duplicate of #2272, and is fixed in PR #2341.
